apps/ledger-live-mobile/src/logic/ratings.tsx

```diff
@@ -80,11 +79,9 @@ const useRatings = () => {
   const ratingsOldRoute = useSelector(ratingsCurrentRouteNameSelector);
   const ratingsHappyMoment = useSelector(ratingsHappyMomentSelector);
   const ratingsDataOfUser = useSelector(ratingsDataOfUserSelector);
-  const accounts: Account[] = useSelector(accountsSelector);
 
-  const accountsWithAmountCount = useMemo(
-    () => accounts.filter(account => account.balance?.gt(0)).length,
-    [accounts],
+  const accountsWithAmountCount = useSelector(
+    accountsWithPositiveBalanceCountSelector,
```

Comment:
With this selector we now have only a re-render (caused by useSelector) every time the _number_ of accounts with a positive balance changes.